_docker/control.rb

```diff
@@ -357,9 +357,6 @@ def initialise_environment(environment, pull_drupal_data_image, system_exec)
   environment = load_environment(tasks)
   initialise_environment(environment, tasks[:docker_pull], system_exec)
 
-  #the docker url is taken from DOCKER_HOST env variable otherwise
-  Docker.url = tasks[:docker] if tasks[:docker]
-
```

Comment:
We're probably okay taking this out as long as people on Macs are using Docker For Mac